server/conf/i18n/messages

```diff
@@ -97,6 +97,49 @@ title.submittedPrograms=Submitted
 toast.applicationSaved=Successfully saved application: application ID {0}
 toast.programCompleted=Application was already completed.
 
+#------------------------------------------------------------------------#
+# TRUSTED INTERMEDIARY DASHBOARD PAGE - text when adding a client #
+#------------------------------------------------------------------------#
+
+banner.clientInfoUpdated=Client info has been successfully updated.
+banner.clientAcctDeleted=The client account for "{0}" has been successfully deleted.
+banner.acctDeleteError=This account cannot be deleted.
+
+button.addNewClient=Add new client
+button.backToEditing=Back to editing
+button.cancel=Cancel
+button.deleteAcct=Delete account
+button.keepAcct=Keep account
+button.nextPage=Next
+button.save=Save
+button.search=Search
+button.viewApplications=View applications
+
+content.numberOfAppSubmitted={0} applications submitted
+content.warning=There are no applications submitted for this client, so no applications will be lost.
+
+dialog.deleteConfirmation=Are you sure you want to delete this client's account?
+
+label.dob=Date of birth
+label.nameOrEmail=Name(s) or email
+label.notes=Notes
+label.contactInfo=Contact Information
+
+link.aboutCiviform=About CiviForm
+link.accessibilityStatement=Accessibility statement
+link.backToClientList=Back to client list
+link.privacyPolicy=Privacy Policy
+
+title.acctSettings=Account settings
+title.allClients=All Clients
+title.clientList=Client list
+title.createClient=Create a new client
+title.displayingAllClients=Displaying all clients
+title.orgMembers=Organization members
+
+toast.asterisk=Required fields are marked with an asterisk
+toast.dobExample=For example: January 18,1986
+
```

Comment:
suggestion (blocking):  With the tweaks of the design this morning, we'll need to add this message too:  `label.nameExample=For example:  Gu or Darren or Darren Gu`.

We're also missing, as far as I can see, "Month", "Day", "Year", "Select", "Clear search", an "Edit" link, "Name", "Email", "Account Status", "Phone number", "New client successfully created.", "Your new client {0} has been added to your client list. You can now begin submitting applications for this client.", "Start an application", "Edit client info", "This account cannot be deleted since it has open applications."